roles/confluent.common/defaults/main.yml

```diff
@@ -16,16 +17,32 @@ confluent_common_repository_redhat_dist_gpgcheck: 1
 confluent_common_repository_redhat_dist_gpgkey: "{{confluent_common_repository_redhat_baseurl}}/{{confluent_repo_version}}/archive.key"
 confluent_common_repository_redhat_dist_enabled: 1
 
+### Boolean to have cp-ansible install Java on hosts
 install_java: true
+
+### Java Package to install on RHEL/Centos hosts
 redhat_java_package_name: java-1.8.0-openjdk
+
+### Java Package to install on Debian hosts
 debian_java_package_name: openjdk-8-jdk
+
+### Java Package to install on Ubuntu hosts
 ubuntu_java_package_name: openjdk-8-jdk
+
+### Deb Repository to use for Java Installation
 ubuntu_java_repository: ppa:openjdk-r/ppa
 
+### Version of Jolokia Agent Jar to Download
 jolokia_version: 1.6.2
+
+### Full URL used for Jolokia Agent Jar Download
 jolokia_jar_url: "http://search.maven.org/remotecontent?filepath=org/jolokia/jolokia-jvm/{{jolokia_version}}/jolokia-jvm-{{jolokia_version}}-agent.jar"
 
+### Full URL used for Prometheus Exporter Jar Download
 jmxexporter_jar_url: https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/0.12.0/jmx_prometheus_javaagent-0.12.0.jar
 
+### Boolean to have cp-ansible download the Confluent CLI
```

Comment:
For variables that have another variable as what they are equal to, should we list out the default in the comment?

Reply:
eh, theres some interesting logic behind this. what do you think is the best way of phrasing it? 
"This variable defaults to whatever rbac_enabled is set to, which by default is false" 

It's important not to just state this defaults to false because if someone turns rbac_enabled on then they should know they are getting the cli.

I kinda think this first iteration is going to have some flaws to it. Like I intentionally omitted some variables. My hope was we can clean them up as we go. wdyt about that plan? Like after this goes thru I want to search for all the TODOs i wrote and move variables out of the defaults dir into the vars dir.

Reply:
Sounds good for a first iteration.  We can always divide up the TODOs as well, to make it less tedious. :D